esp/src/src/nls/es/hpcc.ts

```diff
@@ -487,6 +497,8 @@ export = {
     Lock: "Cerrar",
     log_analysis_1: "log_analysis_1*",
     Log: "Registro",
+    LogAccess_GenericException: "ws_logaccess::GetLogAccessInfo, ha ocurrido una excepción.",
+    LogAccess_LoggingNotConfigured: "No se ha configurado un motor de registro.",
```

Comment:
I'm not privy to the original wording, but as-is this message isn't doesn't make sense to me.
I would just translate as "No se ha configurado LogAccess."